Makefile

```diff
@@ -110,6 +110,10 @@ install-generic:
 	install -m 755 systemd/systemd-openqa-generator "$(DESTDIR)"/usr/lib/systemd/system-generators
 	install -m 644 systemd/tmpfiles-openqa.conf "$(DESTDIR)"/usr/lib/tmpfiles.d/openqa.conf
 	install -m 644 systemd/tmpfiles-openqa-webui.conf "$(DESTDIR)"/usr/lib/tmpfiles.d/openqa-webui.conf
+	install -d -m 755 "$(DESTDIR)"/usr/lib/systemd/system/openqa-webui.service.requires
+	ln -s ../postgresql.service "$(DESTDIR)"/usr/lib/systemd/system/openqa-webui.service.requires/postgresql.service
+	install -d -m 755 "$(DESTDIR)"/usr/lib/systemd/system/openqa-gru.service.requires
+	ln -s ../postgresql.service "$(DESTDIR)"/usr/lib/systemd/system/openqa-gru.service.requires/postgresql.service
```

Comment:
So in the case of remote databases this symlink will not have a target file?

Reply:
Then it won't be installed because the symlink is only part of the openQA-local-db package. So this makes actually sense.

Reply:
However, I'm still wondering about other services. E.g. `openqa-websockets.service` comes in the dependency chain *before* `openqa-webui.service` (`openqa-webui.service` requires `openqa-websockets.service` and not vice verca) and also needs the database. So it should supposedly also be handled by this change (and then openqa-webui.service` *could* be dropped as it then would already indirectly require postgres). The same counts for the other services that currently have `After=postgresql.service`.

Reply:
The link is added by `openQA-local-db` which should be installed in case of remote database, right? So the link shouldn't be there at all.

Reply:
All services have `After=postgresql.service`, but no service actually `Requires=` it and that the source of the problem. Without the require, the ordering is wrong. But as you said, webui requires other services, so if others requires the database, it should theoretically work. I will try that (I did that for the `openqa-gru.service` already where it nicely prevents errors and restarts).